internal/oidc/provider.go

```diff
@@ -1,28 +1,24 @@
 package oidc
 
 import (
-	"crypto/rsa"
-	"fmt"
-
 	"github.com/ory/fosite"
 	"github.com/ory/fosite/compose"
-	"github.com/ory/fosite/token/jwt"
-	"gopkg.in/square/go-jose.v2"
+	"github.com/ory/herodot"
 
 	"github.com/authelia/authelia/internal/configuration/schema"
 	"github.com/authelia/authelia/internal/utils"
 )
 
 // OpenIDConnectProvider for OpenID Connect.
 type OpenIDConnectProvider struct {
-	privateKeys map[string]*rsa.PrivateKey
-
-	Fosite fosite.OAuth2Provider
-	Store  *OpenIDConnectStore
+	Fosite    fosite.OAuth2Provider
+	Store     *OpenIDConnectStore
+	Writer    *herodot.JSONWriter
```

Comment:
Do we really need to introduce this dependency and expose this Writer publicly out of this package?
I'd rather avoid depending on a new lib to serialize JSON unless there is a deep reason behind which would need to be explicited in a comment.